sig-and-wg/sig-core/proposals/installer-release.md

```diff
@@ -0,0 +1,29 @@
+# Kyma Installer with embedded kyma charts and tiller
+
+Created on 2018-09-19 by Piotr Bochyński (@pbochynski).
+
+## Status
+
+Proposed on 2018-09-19
+
+## Abstract
+
+Current installation process involves downloading kyma charts from cloud storage. Such solution allows to release kyma installer and kyma charts independently. In the consequence, there is possibility to install different versions (or flavors) of kyma components using single installer binary. It seems to be a nice feature, but in the reality introduces problems:
+- Build/Release process is more complex because it has to publish kyma charts in cloud storage.
```

Comment:
`Build and release processes are more complex since they require publishing Kyma charts in the cloud storage.` ?